ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -91,6 +96,17 @@ def initialize(opts = {})
       @oidc_settings                    = opts.fetch(:oidc_settings, {})
     end
 
+    # Helper method to set the filename and path for access and error logs
+    def log_filename(value,log_type)
+      if value.nil?
+        prefix = @servername ? "#{@servername}_#{log_type}" : "#{log_type}"
+        suffix = @ssl ? '_ssl.log' : '.log'
+        "#{@logroot}/#{prefix}#{suffix}"
+      else
+        "#{@logroot}/#{value}"
+      end
+    end
```

Comment:
```suggestion
    def log_filename(value,log_type)
      return "#{@logroot}/#{value}" unless value.nil?

      prefix = @servername ? "#{@servername}_#{log_type}" : "#{log_type}"
      suffix = @ssl ? '_ssl.log' : '.log'
      "#{@logroot}/#{prefix}#{suffix}"
    end
```

Reply:
Thanks @treydock  for the suggestion. Didn't have much time today to see the rest. Will do though later in the day and check also the test failures.